MySQLdb/times.py

```diff
@@ -18,15 +18,15 @@
 
 def DateFromTicks(ticks):
     """Convert UNIX ticks into a date instance."""
-    return date(*localtime(ticks)[:3])
+    return date(*gmtime(ticks)[:3])
```

Comment:
I don't want to change behavior from MySQL-python.
Please keep using localtime.